classes/class3/Homework/views/deadcat.handlerbars

```diff
@@ -0,0 +1,6 @@
+<ul>
```

Comment:
This file is a copy of the one above, but with the extension `.handlerbars`. Notice the extra `r` in there. This file is unnecessary and can be deleted. In the future, try not to commit unnecessary and unused files.